agents/heuristics.py

```diff
@@ -0,0 +1,60 @@
+from arguments import get_arguments
+from state_encodings import ENCODING_SCHEMES
+from overcooked_ai_py.mdp.overcooked_mdp import OvercookedState, OvercookedGridworld, Direction, Action
+from overcooked_ai_py.planning.planners import MotionPlanner
+
+from copy import deepcopy
+import json
+import numpy as np
+import os
+import pandas as pd
+from pathlib import Path
+import pickle
+from torch.utils.data import Dataset, DataLoader
+from tqdm import tqdm
+
+class Heuristic():
```

Comment:
All heuristics should return a scalar number for the alternatives they are ranking. In this case, we are ranking possible subtasks, so the output of each heuristic function here should be dictionary where the keys are the subtasks labels and the values are their scalar "worth". (if you prefer, an ordered list of the values is also acceptable, as long as the order is clear).